tools/docker/builder/openwrt/Dockerfile

```diff
@@ -46,68 +62,40 @@ RUN git clone "$OPENWRT_REPOSITORY" openwrt \
 
 WORKDIR /home/openwrt/openwrt
 
-COPY --chown=openwrt:openwrt configs/sdk_${SDK_CONFIG}_config /home/openwrt/sdk_config
+COPY --chown=openwrt:openwrt configs/base_${BASE_CONFIG}_config /home/openwrt/base_config
 
-RUN if [ -n "$INTEL_FEED" ] ; then \
+# We first install from the intel feed preferably, because
+# it replaces some packages from the "packages" feed.
+# If no intel feed is present, nothing will be done so it's still ok.
+RUN echo "src-git prpl ${PRPL_FEED}" >> feeds.conf.default \
+    && if [ -n "$INTEL_FEED" ] ; then \
         echo "src-git intel $INTEL_FEED" >> feeds.conf.default \
         && scripts/feeds update intel \
-        && scripts/feeds install intel_mips ; \
+        && scripts/feeds install intel_mips \
+        && scripts/feeds install -a -p intel ; \
     fi ;\
-    cp /home/openwrt/sdk_config .config \
-    && echo "CONFIG_SDK=y" >> .config \
-    && echo "CONFIG_TARGET_${TARGET}=y" >> .config \
+    scripts/feeds update -a \
+    && scripts/feeds install -a \
+    && cp /home/openwrt/base_config .config \
```

Comment:
HOME is not set as an argument to the Dockerfile, so not accessible here.
We could add it, but since the actual username or path used inside the container don't really matter, I'm not sure it's worth it.